include/vstd/DateUtils.h

```diff
@@ -4,10 +4,8 @@ VCMI_LIB_NAMESPACE_BEGIN
 
 namespace vstd
 {
-
 	DLL_LINKAGE std::string getFormattedDateTime(std::time_t dt);
-	DLL_LINKAGE std::string getDateTimeISO8601Basic(std::time_t dt);
-
+	DLL_LINKAGE std::string getDateTimeFormatted(std::time_t dt);
```

Comment:
What's the difference between `getFormattedDateTime` and `getDateTimeFormatted`?
At least one of them might need better name and probably comment to explain formatting rules

Reply:
I have changed the names to:
getDateTimeLocalized()
getDateTimeFormatted() -> name suggests its a special format, does not give info on the format :( (I would keep it like this until another function arrives)